Application/EdFi.Ods.AdminApi/Features/ClaimSets/ResourceClaimValidator.cs

```diff
@@ -91,6 +105,52 @@ public ResourceClaimValidator()
                 }
                 Validate(dbResourceClaims, dbAuthStrategies, child, resourceClaim.Children, context, claimSetName);
             }
-        }
-    }
+        }
+    }
+
+    private static void ValidateAuthStrategiesOverride<T>(List<string?> dbAuthStrategies, ResourceClaimModel resourceClaim, ValidationContext<T> context, string propertyName)
+    {
+        if (resourceClaim.AuthStrategyOverridesForCRUD.Any())
```

Comment:
Set a validation for null.
`if (resourceClaim.AuthStrategyOverridesForCRUD!=null && resourceClaim.AuthStrategyOverridesForCRUD.Any())`

Reply:
Done